ROADMAP.md

```diff
@@ -23,4 +28,34 @@ SPDX-License-Identifier: Apache-2.0
 - Third-Party Integrations
   - Periodically evaluate and update our existing integrations.
   - Ensure Integrations with CI/CD tools that can leverage Shipwright or existing CI/CD pipelines.
-  - Encourage contributors to create integrations.
\ No newline at end of file
+  - Encourage contributors to create integrations.
+
+## 2025 Release Schedule
+
+The build sub-project releases minor version updates quarterly with new features and
+updates to its dependencies. We use the latest Kubernetes minor version and Tekton LTS version for
+development and testing. Older versions of Kubernetes and Tekton may be supported by the community
+on a best-effort basis.
+
+Below is the tentative release schedule for 2025, with the anticipated versions of Kubernetes and
+Tekton used for development. Actual release dates may vary based on community availability and
+release stability.
+
+- v0.15.0: week of 2025-02-14
+  - Kubernetes version: 1.32
+  - Tekton Pipelines version: 0.68-LTS (expected)
```

Comment:
Do you have insights into if Tekton changed their scheduling? v0.67 should have been released already but is not. I doubt there will be a 0.68 in the next two weeks therefore.

Reply:
0.67 might be delayed due to some significant infra changes happening on their end. They are moving to use `ghcr.io` for future releases as a cost saving measure.

Reply:
I suggest to then state 0.67 here (based on https://tektoncd.slack.com/archives/CLCCEBUMU/p1738001632235419). We probably need to re-check on their future schedule.

Reply:
Ignore it. Tekton skipped 0.67 and went straight to 0.68 - which is an LTS.